image_bld.sh

```diff
@@ -0,0 +1,67 @@
+#! /bin/bash
+
+# This script is uesd to build the different images
+COMP_NAME=$1
+IMAGE_NAME=$2
+IMAGE_LABELS=$3
+DOCKER_MAYBE_CACHE=$4
+IMAGE_STG=$5
+
+# Check the required vars if is defined
+if [ "A${COMP_NAME}" == "A" ];then
+	echo "Which image you want to build,such as anax, agbot or anax-k8s"
+fi
+if [ "A${IMAGE_NAME}" == "A" ];then
+	echo "Please define the image name."
+fi
+if [ "A${IMAGE_LABELS}" == "A" ];then
+	echo "Please define the image labels."
+fi
+if [ "A${DOCKER_MAYBE_CACHE}" == "A" ];then
+	echo "Please define docker cache option."
+fi
+if [ "A${IMAGE_STG}" == "A" ];then
+	echo "Please define the image tag."
+fi
+
+# Special requirements for agbot, anax-k8s ,css-docker, ess-docker image build
+if [[ ${arch} != "amd64" && ${COMP_NAME} == "agbot" ]]; then 
+	echo "agbot image building just support on amd64 !!!"
```

Comment:
Let's clean up the message to use correct English and punctuation on this one and the others below it. I'd suggest something like "Building the agbot image is only supported on amd64 architectures."

Reply:
Great, I will replace it. Thanks